wwwapp/models.py

```diff
@@ -59,6 +59,47 @@ class WorkshopUserProfile(models.Model):
     def __str__(self):
         return '%s: %s, %s' % (self.year, self.user_profile, self.status)
 
+class PESELField(models.CharField):
+    """PESEL field, with checksum verification."""
+
+    def validate(self, pesel: str, user_info: 'UserInfo') -> None:
```

Comment:
This function should probably call super() to validate other builtin constraints too (max length, not empty, not null, etc.)